.github/advanced-commit-linter.yml

```diff
@@ -4,8 +4,8 @@ policy:
       - github: systemd/systemd
     exception:
       note:
-        - rhel-only
-        - RHEL-only
+        - 'rhel-only(: (feature|bugfix|workaround|ci|test|other))?'
+        - 'RHEL-only(: (feature|bugfix|workaround|ci|test|other))?'
```

Comment:
I think the argument should be required.

Reply:
Updated, and I have also added a `doc` for documentation changes.